DancingGoat/Localization/LanguageClient.cs

```diff
@@ -0,0 +1,115 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using KenticoCloud.Delivery;
+using KenticoCloud.Delivery.InlineContentItems;
+using Newtonsoft.Json.Linq;
+
+namespace DancingGoat.Localization
+{
+    public class LanguageClient : IDeliveryClient {
+        public const string DEFAULT_LANGUAGE = "en-us";
+        private readonly DeliveryClient _client;
+        private readonly string _language;
+
+        public LanguageClient(DeliveryClient client, string language)
+        {
+            _client = client;
+            _language = language;
+        }
+
+        public Task<DeliveryItemResponse<T>> GetItemAsync<T>(string codename, params IQueryParameter[] parameters)
+        {
+            return _client.GetItemAsync<T>(codename, new[] {new LanguageParameter(_language)}.Concat(parameters));
+        }
+
+        public Task<DeliveryItemResponse> GetItemAsync(string codename, IEnumerable<IQueryParameter> parameters)
+        {
+            return _client.GetItemAsync(codename, new[] { new LanguageParameter(_language) }.Concat(parameters));
+        }
+
+        public Task<DeliveryItemResponse<T>> GetItemAsync<T>(string codename, IEnumerable<IQueryParameter> parameters = null)
+        {
+            return parameters != null ? _client.GetItemAsync<T>(codename, new[] { new LanguageParameter(_language) }.Concat(parameters)) : 
```

Comment:
This was caused by missing data in sample project. Should be fixed in next commit